chirp/drivers/radtel_rt900.py

```diff
@@ -0,0 +1,1600 @@
+            "30 seconds", "1 minute", "2 minutes", "3 minutes"]
+ALMODE_LIST = ["Site", "Tone", "Code"]
+AUTOLK_LIST = ["Off"] + ABR_LIST[1:4]
+DTMFSPEED_LIST = ["50 ms", "100 ms", "200 ms", "300 ms", "500 ms"]
+DTMFST_LIST = ["Off", "KeyBoard Side Tone", "ANI Side Tone", "KB ST + ANI ST"]
+DUALTX_LIST = ["Off", "A", "B"]
+LANGUAGE_LIST = ["English", "Chinese"]
+MDF_LIST = ["Name", "Frequency", "Channel"]
+MENUQUIT_LIST = ["%s seconds" % x for x in range(5, 55, 5)] + ["60 seconds"]
+OFF1TO9_LIST = ["Off"] + ["%s" % x for x in range(1, 10)]
+PONMSG_LIST = ["Logo", "Voltage"]
+PTTID_LIST = ["Off", "BOT", "EOT", "Both"]
+PTTIDCODE_LIST = ["%s" % x for x in range(1, 16)]
+PTTLT_LIST = ["None", "100 ms"] + \
+             ["%s ms" % x for x in range(200, 1200, 200)]
+QTSAVE_LIST = ["All", "TX", "RX"]
+ROGER_LIST = ["OFF", "BEEP", "TONE1200"]
+RPSTE_LIST = ["Off"] + ["%s ms" % x for x in range(100, 1100, 100)]
+SAVE_LIST = ["Off", "Normal Mode", "Super Mode", "Deep Mode"]
+SCREV_LIST = ["Time (TO)", "Carrier (CO)", "Search (SE)"]
+TONE_LIST = ["1000 Hz", "1450 Hz", "1750 Hz", "2100 Hz"]
+TOT_LIST = ["Off", "15S", "30S", "45S", "60S",
+            "75S"]
+VOXD_LIST = ["%s seconds" % str(x / 10) for x in range(5, 21)]
+WORKMODE_LIST = ["VFO Mode", "Channel Mode"]
+
+
+def get_default_features(self):
+    rf = chirp_common.RadioFeatures()
+    rf.has_settings = True
+    rf.has_bank = False
+    rf.has_ctone = True
+    rf.has_cross = True
+    rf.has_rx_dtcs = True
+    rf.has_tuning_step = False
+    rf.can_odd_split = True
+    rf.has_name = True
+    rf.valid_name_length = 12
+    rf.valid_characters = self._valid_chars
+    rf.valid_skips = ["", "S"]
+    rf.valid_tmodes = ["", "Tone", "TSQL", "DTCS", "Cross"]
+    rf.valid_cross_modes = ["Tone->Tone", "Tone->DTCS", "DTCS->Tone",
+                            "->Tone", "->DTCS", "DTCS->", "DTCS->DTCS"]
+    rf.valid_power_levels = self.POWER_LEVELS
+    rf.valid_duplexes = ["", "-", "+", "split", "off"]
+    rf.valid_modes = ["FM", "NFM"]  # 25 kHz, 12.5 kHz.
+    rf.valid_dtcs_codes = DTCS
+    rf.memory_bounds = (1, self._upper)
+    rf.valid_tuning_steps = [2.5, 5., 6.25, 8.33, 10., 12.5, 20., 25., 50.]
+    rf.valid_bands = self.VALID_BANDS
+    return rf
+
+
+def _enter_programming_mode(radio):
+    serial = radio.pipe
+    mml_jc8810._enter_programming_mode(radio)
+
+    try:
+        ident = serial.read(8)
+        serial.write(radio._cryptsetup)
+        ack = serial.read(1)
+        if ack != CMD_ACK:
+            raise errors.RadioError("Error setting up encryption")
+    except Exception:
+        raise errors.RadioError("Error communicating with radio")
+    if ident not in radio._fingerprint2:
+        LOG.debug(util.hexprint(ident))
+        raise errors.RadioError("Radio returned unknown secondary"
+                                " identification string")
+
+
+def _exit_programming_mode(radio):
+    mml_jc8810._exit_programming_mode(radio)
```

Comment:
...probably don't need this and can just call straight to the other implementation right?